src/content/docs/dropins/payment-services/dictionary.mdx

```diff
@@ -7,5 +7,3 @@ import CodeImport from '@components/CodeImport.astro';
 import CodeInclude from '@components/CodeInclude.astro';
 
 The default Payment Services dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Payment Services drop-in component.
```

Comment:
I believe this is not true yet. Let's double check this before merging. If not, we can remove `dictionary.mdx` entirely for now, I would say.

Reply:
I placed the file for the moment, but I still need to add, or wait to see what we do with this, yes. Thanks!